src/components/textField/index.js

```diff
@@ -621,6 +621,7 @@ export default class TextField extends BaseInput {
         value={value}
         placeholder={placeholderText}
         placeholderTextColor={placeholderColor}
+        selection={Constants.isAndroid && {start: 0}}
```

Comment:
Have you checked this on Android? 
It creates a bug when typing in a TextField

Reply:
I did, but I checked it only in pickers 🙈
Fixed :)